packages/clerk-js/src/ui/components/PricingTable/PricingTableDefault.tsx

```diff
@@ -123,7 +143,7 @@ function Card(props: CardProps) {
         borderWidth: t.borderWidths.$normal,
         borderStyle: t.borderStyles.$solid,
         borderColor: t.colors.$neutralAlpha100,
-        boxShadow: !isCompact ? t.shadows.$cardBoxShadow : undefined,
+        boxShadow: !isCompact ? t.shadows.$cardBoxShadow : t.shadows.$tableBodyShadow,
```

Comment:
Mentioned in slack, but this shadow does not look correct. Seems like design has created a new table shadow that isn't in line with what is in code currently.

Reply:
Looks like, when this table shadow is used, its used as a shadow + border. So the borderWidth defined above should be set to 0.